docs/articles/documentation/reference/command-line-interface.md

```diff
@@ -951,6 +952,19 @@ The `--disable-multiple-windows` option disables support for multi-window testin
 
 *Related configuration file property*: [disableMultipleWindows](configuration-file.md#disablemultiplewindows).
 
+### --retry-test-pages
+
+If this option is enabled, TestCafe retries failed network requests for webpages visited during tests. The retry limit is ten times.
+
+```sh
+testcafe firefox my-tests --retry-test-pages
+```
+
+This feature uses [Service Workers](https://developer.mozilla.org/en-US/docs/Web/API/Service_Worker_API) internally and requires a secure connection.
```

Comment:
Do we need `internally` in this sentence? Which one (do you think) is better:

`This feature uses Service Workers internally and requires a secure connection.`
vs
`This feature uses Service Workers and requires a secure connection.`

Reply:
skip internally